ui/src/views/iam/AddAccount.vue

```diff
@@ -303,7 +303,6 @@ export default {
         } else {
           this.loadMore(apiToCall, page + 1, sema)
         }
-        this.form.domainid = 0
```

Comment:
not sure if it's applicable here - there's also a way to delete this.form.domainid if the key/value need to be removed from the object.

Reply:
@rohityadavcloud don't think we need to remove domainid from the object as API call will be just adding to the existing list so previous selection will still be there